doc/configuration.rst

```diff
@@ -742,6 +742,28 @@ Kubernetes infrastructure key name is ``kubernetes`` in lower case.
 | ``insecure``   | Server should be accessed without verifying the TLS certificate (testing only)  | boolean   | no       |         |
 +----------------+---------------------------------------------------------------------------------+-----------+----------+---------+
 
+.. _option_infra_google:
+
+Google Cloud Platform
+~~~~~~~~~~~~~~~~~~~~~
+
+Google Cloud Platform infrastructure key name is ``google`` in lower case.
+
++-----------------------------+----------------------------------------------+-----------+----------+----------------------------------------+
+|  Option Name                |              Description                     | Data Type | Required | Default                                |
+|                             |                                              |           |          |                                        |
++=============================+==============================================+===========+==========+========================================+
+| ``project``                 | ID of the project to apply any resources to  | string    | yes      |                                        |
++-----------------------------+----------------------------------------------+-----------+----------+----------------------------------------+
+| ``application_credentials`` | Path of file containing credentials*         | string    | no       | Google Application Default Credentials |
```

Comment:
Maybe for a future work but it could be interesting to alternatively provide the ***content*** of this file. This we we can easily use vault to store it.

Reply:
ok ! Adding this alternative configuration parameter allowing to specify the content of a credentials file